### DISCUSSION TITLE
Slider anchors are snapped to DS circles for some reason

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#23531] [Q&A] [answered]
In the distance snap mode, the editor forces slider anchors to fall on those concentric circles. I don't think this is meaningful in the sense of keeping distance equality. It might be a useful feature in some other cases though, like constructing certain geometries in slider shapes, but anyway it should not be in the "distance snap" mode.
![2023-05-13 21 56 24](https://github.com/ppy/osu/assets/33959010/0e07560c-056a-4f0c-95af-4366614847ec)

**Comment by peppy** [comment, marked as answer]
#23662
